packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -33,6 +32,7 @@ export interface AbstractMessage {
   type: string;
   rawSig?: string;
   securityProviderResponse?: Map<string, Json>;
+  error?: string;
```

Comment:
As discussed, I don't think we're using this now so can we remove this property?